tests/driver_l3gxxxx/app.config.test

```diff
@@ -0,0 +1,2 @@
+# application configuration. This is only needed during migration.
+CONFIG_MODULE_L3GXXXX=y
```

Comment:
```suggestion
CONFIG_MODULE_L3GXXXX=y
CONFIG_MODULE_L3GD20H=y
```

Reply:
Wouldn't this configure the L3GD20H permanently as a sensor variant? What if a board has another sensor variant like [stm32f429i-disc1](https://github.com/RIOT-OS/RIOT/blob/461f9b103e630d5a69162158980f2edd683eb350/boards/stm32f429i-disc1/Kconfig#L33). The test application enables module `l3gd20h` by default if no sensor variant is defined.

Reply:
> Wouldn't this configure the L3GD20H permanently as a sensor variant?
Yes, it would.

> What if a board has another sensor variant like [stm32f429i-disc1](https://github.com/RIOT-OS/RIOT/blob/461f9b103e630d5a69162158980f2edd683eb350/boards/stm32f429i-disc1/Kconfig#L33). The test application enables module `l3gd20h` by default if no sensor variant is defined.

Ok, I missed this check in the `Makefile`. Would this match the 'automatic' variant selection in the driver Kconfig (based on the features)? In that case this is fine.

Reply:
The [default selection of the variant](https://github.com/RIOT-OS/RIOT/blob/8e7ea65fbf3310d17142153a49e7d00e6ca25f51/drivers/l3gxxxx/Kconfig#L53) in `Kconfig` depends on on the variant a board definition has. I guess the first config entry of the choice is used if the board doesn't define a variant which would be L3GD20H, right?

Reply:
Yes, if none of the `default`s could be applied the first valid option is the default instead.